src/disputables/index.js

```diff
@@ -190,6 +192,27 @@ function getTerminalActions(transactions) {
     .flat()
 }
 
+/**
+ * Gets execution path specified by `transactions`
+ * @param {Array} transactions Array of transactions in the execution path
+ * @param {Array} apps Array of all the org's apps
+ * @returns {Array} Execution path
+ */
+function contexctualizeExecutionPath(transactions, apps) {
+  return transactions.map(transaction => {
+    const app = apps.find(app => addressesEqual(app.address, transaction.to))
+    const appName = app ? describeAppName(app.name) : 'Unknown app'
```

Comment:
To double check, we are only using the list of installed applications (from Connect) for this, right? If someone, for some reason, uninstalled an app later on, it would appear as "Unknown app"?

I think this is fine, but I'll keep a mental note about it and also ask the Connect team about making sure we can detect uninstalled apps.

Reply:
> To double check, we are only using the list of installed applications (from Connect) for this, right? If someone, for some reason, uninstalled an app later on, it would appear as "Unknown app"?

Yes that's correct.

> I think this is fine, but I'll keep a mental note about it and also ask the Connect team about making sure we can detect uninstalled apps.

Cool, we can add a comment and ask connect team yes :)